dotCMS/src/main/java/com/dotcms/aspects/interceptors/CloseDBMethodInterceptor.java

```diff
@@ -0,0 +1,38 @@
+package com.dotcms.aspects.interceptors;
+
+import static com.dotcms.util.AnnotationUtils.getMethodAnnotation;
+
+import com.dotcms.aspects.DelegateMethodInvocation;
+import com.dotcms.aspects.MethodInterceptor;
+import com.dotcms.business.CloseDB;
+
+
+import com.dotmarketing.db.DbConnectionFactory;
+
+/**
+ * Method handler for the {@link CloseDBIfOpened} annotation aspect
+ * @author jsanca
+ */
+public class CloseDBMethodInterceptor implements MethodInterceptor<Object> {
+
+    public static final CloseDBMethodInterceptor INSTANCE = new CloseDBMethodInterceptor();
+
+
+    @Override
+    public Object invoke(final DelegateMethodInvocation<Object> delegate) throws Throwable {
+
+
+        final CloseDB closeDB =
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "closeDB". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "closeDB" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)